tools/init_blockchain.py

```diff
@@ -42,5 +42,6 @@ def create_and_distribute_token(
         )
 
     for receiver in receivers:
-        token_proxy.transact('transfer', receiver, amount_per_receiver)
+        startgas = 100000
```

Comment:
use the constant I told you to create before here too

Reply:
I avoided declaring this constant when it's only used in the tests. If you want I can create a constant, but this value is completely arbitrary and I just set it to satisfy the API call so that the fixture would execute properly.

Reply:
would be nice for consistency since it's used in multiple places. I can see it's only used in tests but it's nice to be consistent.